src/world.rs

```diff
@@ -12,6 +11,9 @@ use std::time::Duration;
 
 /// Tracks all the state for the simulated world.
 pub(crate) struct World {
+    /// Defines the available address range of the simulation.
+    pub(crate) ip_net: IpNetwork,
```

Comment:
Related to my previous comment: this is not really a network, it's a subnet, and turmoil doesn't support >1 subnet.